SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -124,14 +150,12 @@ elif [[ "$REPO_TYPE" == "eos" ]] && [[ "$REPOSITORY" =~ $REPO_EOS_PATTERN ]]; th
     ENV_NAME="$(basename $REPO_PATH)_env"
 
     if [ ! -d "${REPO_PATH}" ]; then
-        _log "ERROR: Invalid ${REPO_TYPE} repository (${REPO_PATH})." && _log
-        exit 1
+        _error "Invalid ${REPO_TYPE} repository (${REPO_PATH})."
```

Comment:
The changes related to this new `_error` function are not related to conda, are they? They should be in a separate commit.